src/app/search-courses/search-courses.component.html

```diff
@@ -7,5 +7,5 @@
     >
     <button (click)="onFindClick()">Search</button>
   </div>
-  <button>Add Course</button>
+  <button (click)="onClickAddCourse()">Add Course</button>
```

Comment:
Maybe it would be more suitable to use router directive here instead of handling the click event

Reply:
Fixed